src/app/my-plan/_components/client/plan-filter-section.tsx

```
@@ -213,15 +224,22 @@ const PlanFilterSection = ({
-                key={plan.planId}
-                plan={plan}
-                nickname={userNickname}
-                onEdit={handleEdit}
-                onDelete={() => handleDelete(plan.planId)}
-              />
+              <>
+                <div className="w-[calc(50%-6px)] md:hidden">
+                  <PlanVerticalCard key={plan.planId} plan={plan} />
+                </div>
+                <div className="hidden md:block">
+                  <PlanHorizontalCard
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)